tests/integration/security/testdata/requestauthn/f-authn.yaml.tmpl

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
+metadata:
+  name: "request-authn-for-f"
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: f
+  jwtRules:
+  - issuer: "test-issuer-1@istio.io"
+    jwksUri: "https://raw.githubusercontent.com/istio/istio/master/tests/common/jwt/jwksfake.json"
```

Comment:
Add a comment that this file is expected to not exist, better to rename the `jwksfake.json` to `jwks-not-exist.json`.

Reply:
Sure. Thanks for pointing it out.